governance/polls/templates/Generic/$GenericApprovalWithoutMajority - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,57 @@
+---
+title: $GenericChange - $date_MONTH_DD,_YYYY
+summary: Signal your support or opposition to $GenericChange.
+discussion_link: $discussion_link
+parameters:
+  input_format:
+    type: choose-free
+    options: [1,2]
```

Comment:
Approval-without-majority with two options is just a plurality poll. Voting for both or neither is a non-vote, voting for one and not the other = plurality.

Reply:
This was an oversight. I have added a third option to the template.